pkg/apis/config/types.go

```diff
@@ -41,11 +41,18 @@ type CoschedulingArgs struct {
 // ModeType is a "string" type.
 type ModeType string
 
+type TopologyManagerPolicy string
+
 const (
 	// Least is the string "Least".
 	Least ModeType = "Least"
 	// Most is the string "Most".
 	Most ModeType = "Most"
+
+	// to preserve consistency keep it in pkg/apis/core/types.go"
+	SingleNUMANodeTopologyManagerPolicy TopologyManagerPolicy = "SingleNUMANode"
```

Comment:
these constants (all of type TopologyManagerPolicy) represent behavior of the resource management component on the node (now it's TopologyManager, but it could be some external resource manager). This patch series adds support only for SingleNUMANode (it's single-numa-node policy of the TopologyManager) and PodTopologyScope (it's pod level resource counting in TopologyManager). In CRD it's array of string.  So here could be a string too or we can keep TopologyManagerPolicy type in noderesourcetopology-api.

Regarding name of the type, since it represent behavior/policy of the node's resource management based on hardware topology. I think names like TopologyPolicy/TopologyManagerPolicy/ResourceTopologyPolicy show the purpose.

Reply:
It sounds to be we should name them as `SingleNUMANodeGeneral` and `SingleNUMANodePodTopology`?

Reply:
I agree, it worth changing it now, since current names have some historical aspects )